_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -34,7 +35,7 @@ This rule applies to each [link history state][] of any [semantic link][], for w
 For each test target, there exists at least one [adequate inspection state][] and at least one [visible][] [inclusive descendant][] or [ancestor][] element that is not an [ancestor][] of the **non-link line text**, such that at least one of the following is true:
 
 - **distinguishing style**: the [visible][] [inclusive descendant][] or the [ancestor][] element has a [distinguishable style][] from each element containing **non-link line text**; or
-- **distinguishing content**: the [visible][] [inclusive descendant][] has adjacent content (such as an image or text) that indicates the test target is a link.
+- **distinguishing content**: the [text nodes][text node] of the [visible][] [inclusive descendant][] or the [text nodes][text node] [visible][] of the [inclusive sibling][] include the word "link" (or a translation of "link" to the [default page language][]).
```

Comment:
minor point: should be the language of the element, not of the page.
```html
<html lang="fr>
<p lang="en">link: <a href="…>Foo</a></p>
</html>
```

Reply:
This has been solved by removing the distinguishable content from the expectation